version.php

```diff
@@ -28,8 +28,9 @@
     die('Direct access to this script is forbidden.'); // It must be included from a Moodle page.
 }
 
-$plugin->version   = 2020051200;
-$plugin->release   = 2020051200;
+$plugin->version   = 2020051201;
+$plugin->release   = 2020051201;
 $plugin->maturity  = MATURITY_STABLE;
 $plugin->requires  = 2014051200; // Moodle 2.7 release and upwards.
 $plugin->component = 'tool_lockstats';
+$plugin->incompatible = 310;
```

Comment:
$plugin->incompatible is used to specify the lowest working version, this line here would mean that this plugin could be installed on 3.11+ only.

This line would best be added to the 3.10 branch, and equal the value 39.

See the documentation here, https://docs.moodle.org/dev/version.php

Reply:
This is from the docs

" Specifies the earliest incompatible Moodle major branch.Any Moodle versions including and above this will be prevented from installing the plugin, and a message will be given when attempting installation.

$plugin->incompatible = 36; // Moodle 3.6.x and later are incompatible.
--"

So 3.10 and higher are incompatible

Reply:
Apologies, I confused myself with the code and documentation. This line https://github.com/moodle/moodle/blob/MOODLE_39_STABLE/lib/upgradelib.php#L617 states the inverse of the documentation. A tracker exists to fix this https://tracker.moodle.org/browse/MDL-72324

I think for the moment, please remove this line, as the tests will always fail. Once Moodle has updated, we can restore the incompatibility check.